x-pack/plugin/eql/src/main/antlr/EqlBase.g4

```diff
@@ -84,11 +84,15 @@ booleanExpression
 
 
 valueExpression
-    : primaryExpression predicate?                                                      #valueExpressionDefault
-    | operator=(MINUS | PLUS) valueExpression                                           #arithmeticUnary
-    | left=valueExpression operator=(ASTERISK | SLASH | PERCENT) right=valueExpression  #arithmeticBinary
-    | left=valueExpression operator=(PLUS | MINUS) right=valueExpression                #arithmeticBinary
-    | left=valueExpression comparisonOperator right=valueExpression                     #comparison
+    : defaultExpression                                                                     #valueExpressionDefault
+    | left=defaultExpression comparisonOperator right=defaultExpression                     #comparison
+    ;
+
+defaultExpression
```

Comment:
Please let me know of a better suggestion for this name as I really couldn't come up with one.

Reply:
looks like these are all arithmetic, so maybe `arithmeticExpression`?

Reply:
I opt for operators (`operatorExpression`) since the block contains arithmetic and comparison operators, the latter being logical not arithmetic.